### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by MatthijsBurgh**
remove roslib dependency
cleanup CMakeLists.txt and package.xml
add missing dep on code_profiler

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,18 +35,18 @@ We assume you have successfully installed ROS and set-up a Catkin workspace. Che
...
+    git clone https://github.com/tue-robotics/geolib2
+    git clone https://github.com/tue-robotics/rgbd
     git clone https://github.com/tue-robotics/tue_config.git
-    git clone https://github.com/tue-robotics/ed_object_models.git
+    git clone https://github.com/tue-robotics/tue_filesystem
     git clone https://github.com/tue-robotics/tue_serialization
-    git clone https://github.com/tue-robotics/rgbd
 
-You will also need the following system dependencies (ROS Indigo, Ubuntu 14.04):
+You will also need the following system dependencies (ROS kinetic, Ubuntu 14.04):
```
Comment (ar13pit) [inline README.md]: Ubuntu 16.04

File: package.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<package>
+<package format="2">
   <name>ed</name>
   <version>0.0.0</version>
```
Comment (ar13pit) [inline package.xml]: Maybe a version bump

Comment (MatthijsBurgh) [reply package.xml]: To what? :D

Comment (ar13pit) [reply package.xml]: 1.0.0 :D

Comment (MatthijsBurgh) [reply package.xml]: No it isn't good enough for a 1.X

**Pull request merged**